src/tasks.js

```diff
@@ -61,29 +62,58 @@ if(config.production) {
 		 */
 
 		later.setInterval(() => {
-			console.log('[' + new Date() + '] Update Weather');
+			console.log(`[${new Date()}] Update Weather`);
 
 			weather.update(err => {
 				if (err) {
-					console.log('[' + new Date() + '] Error occured for weather! (' + err + ')');
+					console.log(`[${new Date()}] Error occured for weather! (${err})`);
 
 					// Alert admins if problem getting weather
 					admins.sendEmail(db, {
 						subject: 'Error Notification - Weather Retrieval',
 						html: 'There was an error when retrieving the weather.<br>Error message: ' + err
 					}, err => {
 						if (err) {
-							console.log('[' + new Date() + '] Error occured when sending admin error notifications! (' + err + ')');
+							console.log(`[${new Date()}] Error occured when sending admin error notifications! (${err})`);
 							return;
 						}
-						console.log('[' + new Date() + '] Alerted admins of error! (' + err + ')');
+						console.log(`[${new Date()}] Alerted admins of error! (${err})`);
 					});
 				} else {
-					console.log('[' + new Date() + '] Successfully updated weather!');
+					console.log(`[${new Date()}] Successfully updated weather!`);
 				}
 			});
 
 		}, fiveMinuteInterval);
+
```

Comment:
We should also add a task to update everyone's Canvas cache. We might need to discuss how often this will be, though.